code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -245,30 +269,29 @@
 
 /mob/living/simple_animal/proc/handle_atmos(var/atmos_suitable = 1)
 	//Atmos
-
 	if(!loc)
 		return
 
+	// don't bother checking it twice if we got a supplied 0 val.
```

Comment:
Sort of a nitpick but this comment is about checking `atmos_suitable`, not `environment`, isn't it? Shouldn't it be next to that line?